File: src/class/GestorActualizaciones.js
export default class Gestor {
  constructor(bodega, vino, enofilos, varietal, tipoDeUva, maridaje) {
    this.bodega = bodega; // Relación con Bodega
    this.bodegaSeleccioanada = bodegaSeleccionada;
    this.vino = vino; // Relación con Vino
    this.enofilos = enofilos; // Relación con Enofilo
    this.varietal = varietal; // Relación con Varietal
    this.tipoDeUva = tipoDeUva; // Relación con TipoDeUva
    this.maridaje = maridaje; // Relación con Maridaje
  }

  // Métodos getter y setter para las relaciones de Gestor
  get bodega() {
    return this.bodega;
  }

  set bodega(bodega) {
    this.bodega = bodega;
  }

  get vino() {
    return this.vino;
  }

  set vino(vino) {
    this.vino = vino;
  }

  get enofilo() {
    return this.enofilo;
  }

  set enofilo(enofilo) {
    this.enofilo = enofilo;
  }

  get varietal() {
    return this.varietal;
  }

  set varietal(varietal) {
    this.varietal = varietal;
  }

  get tipoDeUva() {
    return this.tipoDeUva;
  }

  set tipoDeUva(tipoDeUva) {
    this.tipoDeUva = tipoDeUva;
  }

  get maridaje() {
    return this.maridaje;
  }

  set maridaje(maridaje) {
    this.maridaje = maridaje;
  }
}

File: src/com/BonVino/proyect/GestorActualizaciones.java
package com.BonVino.proyect;

public class GestorActualizaciones {
    private Bodega bodega;
    private Bodega bodegaSeleccionada;
    private Vino vino;
    private Enofilo enofilo;
    private Varietal varietal;
    private TipoDeUva tipoDeUva;
    private Maridaje maridaje;

    // Constructor
    public GestorActualizaciones(Bodega bodega, Bodega bodegaSeleccionada, Vino vino, Enofilo enofilo,
                  Varietal varietal, TipoDeUva tipoDeUva, Maridaje maridaje) {
        this.bodega = bodega;
        this.bodegaSeleccionada = bodegaSeleccionada;
        this.vino = vino;
        this.enofilo = enofilo;
        this.varietal = varietal;
        this.tipoDeUva = tipoDeUva;
        this.maridaje = maridaje;
    }

    // Métodos getter y setter para las relaciones de Gestor
    public Bodega getBodega() {
        return bodega;
    }

    public void setBodega(Bodega bodega) {
        this.bodega = bodega;
    }

    public Bodega getBodegaSeleccionada() {
        return bodegaSeleccionada;
    }

    public void setBodegaSeleccionada(Bodega bodegaSeleccionada) {
        this.bodegaSeleccionada = bodegaSeleccionada;
    }

    public Vino getVino() {
        return vino;
    }

    public void setVino(Vino vino) {
        this.vino = vino;
    }

    public Enofilo getEnofilo() {
        return enofilo;
    }

    public void setEnofilo(Enofilo enofilo) {
        this.enofilo = enofilo;
    }

    public Varietal getVarietal() {
        return varietal;
    }

    public void setVarietal(Varietal varietal) {
        this.varietal = varietal;
    }

    public TipoDeUva getTipoDeUva() {
        return tipoDeUva;
    }

    public void setTipoDeUva(TipoDeUva tipoDeUva) {
        this.tipoDeUva = tipoDeUva;
    }

    public Maridaje getMaridaje() {
        return maridaje;
    }

    public void setMaridaje(Maridaje maridaje) {
        this.maridaje = maridaje;
    }
}
